src/index.css

```diff
@@ -2,34 +2,15 @@
 	--color-black: hsla(220, 13%, 18%, 1);
 	--color-gray-dark: hsla(220, 13%, 25%, 1);
 	--color-white: hsla(220, 13%, 98%, 1);
-	--color-gray-light: hsla(220, 13%, 94%, 1);
-	--color-emerald-green: hsla(168, 92%, 25%, 1);
-	--color-vermillion-green: hsla(168, 92%, 43%, 1);
-	--color-cobalt-blue: hsla(215, 100%, 34%, 1);
-	--color-pastel-blue: hsla(215, 100%, 73%, 1);
-	--color-red: hsl(0, 68%, 42%);
+	--color-dark-blue-two: #1c478b;
+	--color-powder-blue: #accbff;
 
-	--color-accent: var(--color-pastel-blue);
-	--color-bg: var(--color-black);
-	--color-border: hsla(220, 13%, 32%, 1);
-	--color-error: var(--color-red);
-	--color-text: var(--color-white);
-}
-
-@media screen and (prefers-color-scheme: light) {
-	:root {
-		--color-accent: var(--color-cobalt-blue);
-		--color-bg: var(--color-white);
-		--color-border: hsla(220, 13%, 78%, 1);
-		--color-text: var(--color-black);
-	}
-}
-
-:root.theme-light {
-	--color-accent: var(--color-cobalt-blue);
-	--color-bg: var(--color-white);
-	--color-border: hsla(220, 13%, 78%, 1);
-	--color-text: var(--color-black);
+	--color-bg-dark: var(--color-black);
+	--color-bg-light: var(--color-white);
```

Comment:
I spoke to Tim because I was having a hard time figuring out the transparency issues with the modal. If you comment out line 8 and 9 it resolves it, BUT it creates a whole set of other styling issues with your dark mode. 
![Screenshot 2024-04-03 at 5 08 54 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/97368143/90d778c1-9b20-447f-8513-769de5fbbe8a)